index.js

```diff
@@ -51,7 +49,7 @@ var createSourceMapLocatorPreprocessor = function(args, logger, helper) {
     if (!mapUrl) {
       fileMap(file.path + ".map");
     } else if (/^data:application\/json/.test(mapUrl)) {
-      inlineMap(mapUrl);
+      inlineMap(mapUrl.slice('data:application/json'.length));
```

Comment:
The start was only used in the regexp in `inlineMap`, so not interesting and can be discarded